### ISSUE TITLE
Implement Kitty Extended Terminal Protocols

### ISSUE THREAD

**Issue by nyabinary** [pop-os/cosmic-term#39]
Would be nice for the terminal to support all the extensions Kitty has:
https://sw.kovidgoyal.net/kitty/protocol-extensions/

**Comment by nyabinary** [comment]
> Especially [underlines](https://sw.kovidgoyal.net/kitty/underlines/), it's supported by many other terminals and very useful!
Yep, the Helix Editor depends on some Kitty Protocols for their features, and one of those are underlines:
https://github.com/helix-editor/helix/wiki/Terminal-Support

**Comment by DrunkenAlcoholic** [comment]
Would really like to see the kitty graphics protocol supported in cosmic-term
https://sw.kovidgoyal.net/kitty/graphics-protocol/
